test/minitest_runner.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.unshift(__dir__)
+
+require 'minitest'
```

Comment:
Can you use "test-unit" instead of `minitest` ?

`ruby/ruby` migrates `minitest` to `test-unit` in this year.

Reply:
@hsbt 
Could you tell me why `ruby/ruby` migrates `minitest` to `test-unit`?

Reply:
see https://github.com/rubygems/rubygems/pull/3141

Basically, `ruby`ruby` uses `test-unit`, not `minitest`. rubygems and psych only uses minitest namespace. I did migrate them to `test-unit` in May, 2021.

Reply:
Thank you!

Reply:
6d42955